.github/workflows/ci.yml

```diff
@@ -61,21 +61,14 @@ jobs:
         with:
           go-version-file: systemtest/go.mod
           cache: true
+          cache-dependency-path: |
+            go.sum
+            systemtest/go.sum
       - run: docker compose up -d
       - env:
           GOTESTFLAGS: "-v -json"
-          OUTPUT_JSON_FILE: "build/TEST-go-system_tests.out.json"
-          OUTPUT_JUNIT_FILE: "build/TEST-go-system_tests.xml"
           GH_TOKEN: ${{ github.token }}
-        run: |
-          go run -modfile=tools/go.mod gotest.tools/gotestsum \
```

Comment:
i'm not aware that we need the jsonfile and junitfile but pinging @v1v for :eyes:

Reply:
That's something used in https://github.com/elastic/apm-server/blob/main/.github/workflows/test-reporter.yml

But if you have not used it, then it seems it's not needed :)

Reply:
Actually, it is disabled 

<img width="1557" alt="image" src="https://github.com/user-attachments/assets/949199d5-af32-4698-a4a5-0a30c65ef2f1" />

That's the reason, you didn't use it :/